bundles/org.openhab.core.model.item.ide/pom.xml

```diff
@@ -14,6 +14,11 @@
 
   <name>openHAB Core :: Bundles :: Model Item IDE</name>
 
+  <properties>
+    <maven.compiler.source>17</maven.compiler.source>
+    <maven.compiler.target>17</maven.compiler.target>
+  </properties>
```

Comment:
Does it also work if you use `<maven.compiler.release>17</maven.compiler.release>` instead?

Is it also possible to add the property to the [model-specific-profile](https://github.com/openhab/openhab-core/blob/be9cbf26491f976272c361e73292336146026adb/bundles/pom.xml#L272-L356) instead?

Reply:
I will try ```<maven.compiler.release>17</maven.compiler.release>```...

```model-specific-profile``` would be way more elegant, though it covers more directories.

> org.openhab.core.model.codegen
> org.openhab.core.model.core
> org.openhab.core.model.item.runtime
> org.openhab.core.model.lsp
> org.openhab.core.model.persistence.runtime
> org.openhab.core.model.rule.runtime
> org.openhab.core.model.script.runtime
> org.openhab.core.model.sitemap.runtime
> org.openhab.core.model.thing.runtime

Will check.